js/plugins/util.js

```diff
@@ -0,0 +1,32 @@
+var util = {
```

Comment:
This name seems like it might easily cause a conflict with another global util variable. Is it possible to namespace this in `clinicalTimeline.util`?